README.md

```diff
@@ -18,6 +18,8 @@ Please prefer filing an issue in one of them if your feature request or bug repo
 - [godo](https://github.com/digitalocean/godo): The official DigitalOcean Go API client to support programmatic access to DOKS.
 - [kops](https://github.com/kubernetes/kops): kops helps you create, destroy, upgrade, and maintain Kubernetes clusters from the command line. It comes with initial support for running DIY Kubernetes clusters on DigitalOcean ([tutorial](https://github.com/kubernetes/kops/blob/master/docs/tutorial/digitalocean.md)).
 - [Cluster Management API implementation for DigitalOcean](https://github.com/kubernetes-sigs/cluster-api-provider-digitalocean): Supports managing Kubernetes clusters on DigitalOcean through the [Cluster Management API](https://github.com/kubernetes-sigs/cluster-api).
+- [Kubernetes autoscaler](https://github.com/kubernetes/autoscaler): Autoscaling-related components for Kubernetes, including support for DigitalOcean DOKS.
```

Comment:
We maintain a fork of this if you want to link to that instead? Not sure what makes the most sense, we do keep it in sync with upstream.

Reply:
Pushed a change to mention the fork as well. WDYT?